src/views/DataExploration/utils/OntologyTree.tsx

```diff
@@ -59,8 +59,8 @@ export const searchTree = (element: TreeNode, matchingTitle: string): TreeNode |
   if (element.title === matchingTitle) {
     return element;
   } else if (element.children != null) {
-    var i;
-    var result = null;
+    let i;
```

Comment:
Pourquoi ne pas faire le let directement dans le for ?
for (**let** i = 0; result == null && i < element.children.length; i++)

Reply:
c'est le linter qui aime pas les `var` et les a change en `let`. Jai pas vraiment touche a ce bout de code. Mais oui le `let i` je peux l'enlever.